tools/hangar/__snapshots__/invalid.ts.snap

```diff
@@ -899,6 +899,20 @@ error: Expected type to be optional, but got \\"bool\\" instead
    |            ^^^^ Expected type to be optional, but got \\"bool\\" instead
 
 
+error: Property access unsupported on type \\"A?\\"
+   --> ../../../examples/tests/invalid/optionals.w:61:9
+   |
+61 | let c = b.a.val;
+   |         ^^^ Property access unsupported on type \\"A?\\"
```

Comment:
Is that error message intentional?

Reply:
@staycoolcall911 Actually this error message is current issue with optional types that gets resolved in the sister PR for optional chaining: https://github.com/winglang/wing/pull/2663

However I cannot produce my intended error without producing this one as well.